ui/page/send/page.go

```diff
@@ -408,23 +405,23 @@ func (pg *Page) constructTx() {
 		pg.feeRateSelector.USDExchangeSet = true
 		pg.txFeeUSD = fmt.Sprintf("$%.4f", utils.CryptoToUSD(pg.exchangeRate, feeAndSize.Fee.CoinValue))
 		pg.feeRateSelector.TxFeeUSD = pg.txFeeUSD
-		pg.totalCostUSD = utils.FormatUSDBalance(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, totalSendingAmount.ToCoin()))
-		pg.balanceAfterSendUSD = utils.FormatUSDBalance(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, balanceAfterSend.ToCoin()))
+		pg.totalCostUSD = utils.FormatAsUSD(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, totalSendingAmount.ToCoin()))
+		pg.balanceAfterSendUSD = utils.FormatAsUSD(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, balanceAfterSend.ToCoin()))
 
 		usdAmount := utils.CryptoToUSD(pg.exchangeRate, wal.ToAmount(amountAtom).ToCoin())
-		pg.sendAmountUSD = utils.FormatUSDBalance(pg.Printer, usdAmount)
+		pg.sendAmountUSD = utils.FormatAsUSD(pg.Printer, usdAmount)
 	}
 }
 
-func (pg *Page) showBalaceAfterSend() {
+func (pg *Page) showBalanceAfterSend() {
 	if pg.sourceAccountSelector != nil {
 		sourceAccount := pg.sourceAccountSelector.SelectedAccount()
 		if sourceAccount.Balance == nil {
 			return
 		}
 		balanceAfterSend := sourceAccount.Balance.Spendable
 		pg.balanceAfterSend = balanceAfterSend.String()
-		pg.balanceAfterSendUSD = utils.FormatUSDBalance(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, balanceAfterSend.ToCoin()))
+		pg.balanceAfterSendUSD = utils.FormatAsUSD(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, balanceAfterSend.ToCoin()))
```

Comment:
Why was the function name changed, the function took a USD balance and formatted it

Reply:
It was previously limited(in the naming) which is a bit confusing for a generic helper function that's not always used to format a "balance".

Reply:
Are there instances where it was used tl format things other than a balance? Please link me

Reply:
> Are there instances where it was used tl format things other than a balance? Please link me

Not really, but it's now used in the market section in this PR.

Reply:
`FormatAsUSD` doesn't look clear to me.

If the naming concern is due to the function being used to format the price on the market overview, then i think it can fly

Reply:
> FormatAsUSD doesn't look clear to me.

I think I should add "String", yh?